database.yml

```diff
@@ -0,0 +1,18 @@
+default: &default
+  adapter: postgresql
+  pool: 5
+  timeout: 5000
+  encoding: unicode
+
+development:
+  <<: *default
+  database: slack_ruby_bot_server_development
+  username: slack_ruby_bot_server
```

Comment:
I don't think this is how PostgreSQL default config works, we want this code to be able to run PostgreSQL tests without any changes.